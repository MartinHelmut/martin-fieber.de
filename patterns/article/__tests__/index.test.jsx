import "jest-styled-components";
import React from "react";
import renderer from "react-test-renderer";

import Article from "../index";

describe("Article", () => {
  test("should render", () => {
    const tree = renderer
      .create(<Article>An interesting article</Article>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
