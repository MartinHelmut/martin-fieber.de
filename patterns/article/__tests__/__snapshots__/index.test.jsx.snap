// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article should render 1`] = `
.c0 {
  border-bottom: 1px solid var(--color-grey-500);
  padding: var(--size-500) 0;
  margin: 0 0 var(--size-500) 0;
}

.c0:last-child {
  border: 0;
  margin: 0;
}

<article
  className="c0"
>
  An interesting article
</article>
`;
